### ISSUE TITLE
Load account progress from blockchain (MyAccountScreen)

### ISSUE THREAD

**Issue by marcelomorgado** [tasitlabs/tasit#318]
The `MyAccountScreen` is looking to `redux` state to show creation progress, use already wrote code (from helpers/index.js > restoreCreationStateOfAccountFromBlockchain) to check progress from the blockchain.

Should this screen update the redux state (same logic as MyAssetsScreen)?

**Comment by pcowgill** [comment]
> Should this screen update the redux state (same logic as MyAssetsScreen)?

Yeah, I think so, if what we get differs from what we have stored locally. Perhaps we can even share some logic with the other screen on how to handle this "diff-ing"

**Comment by pcowgill** [comment]
I'll note that we'll continue to show the tx links using the `actionId`'s that we originally persisted when sending the Tasit actions, and we don't yet want to go down the rabbit hole of finding those tx from the blockchain using the addresses we know.

So when we persist the local account state based on the state of the blockchain, let's be sure not to overrwite `actionId`'s
